sru/release-27/test_trusty_upgrade_to_xenial.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+# Test if upgrading from trusty machine already attached to xenial will keep
+# services still enabled after the do-release-upgrade operation. Currently,
+# this will not happen because the latest version of uaclient in trusty does
+# not enable do-release-upgrade to run a post script that handles the services
+# migration when performing an upgrade.
+# This script is adding a solution for that problem by manually adding the necessary
+# configs for do-release-upgrade (This configurations are done for release 26 forward)
+set -x
+
+series=trusty
+name=$series-upgrade-lxd
+
+function fix_upgrade_for_esm_infra() {
+    multipass exec $name -- sh -c "echo 'Pockets=security,updates,proposed,backports,infra-security,infra-updates,apps-security,apps-updates' >> allow.cfg"
+    multipass exec $name -- sh -c "echo '[Distro]\nPostInstallScripts=./xorg_fix_proprietary.py, /usr/lib/ubuntu-advantage/upgrade_lts_contract.py' >> allow.cfg"
+}
+
+
+multipass delete $name
+multipass purge
+multipass launch $series --name $name
+
+multipass exec $name -- sudo apt-get upgrade -y
+multipass exec $name -- sudo apt-get dist-upgrade -y
+multipass exec $name -- ua version
+multipass exec $name -- sudo ua attach $UACLIENT_BEHAVE_CONTRACT_TOKEN
+multipass exec $name -- sudo ua status
+multipass exec $name -- sudo add-apt-repository ppa:orndorffgrant/uaclient-staging-27 -y
```

Comment:
I think we should change this to actually setup -proposed on the system under test

Something like what we do for cloud-init (but installing ubuntu-advantage-tools instead) https://github.com/cloud-init/ubuntu-sru/blob/main/sru-scripts/sru-vars.template#L59-L76